argilla-server/src/argilla_server/api/schemas/v1/records.py

```diff
@@ -82,13 +83,21 @@ class Config:
 
 
 class RecordCreate(BaseModel):
-    fields: Dict[str, Union[StrictStr, None, List[Dict[StrictStr, StrictStr]]]]
+    fields: Dict[str, Union[List[ChatFieldValue], StrictStr, None]]
     metadata: Optional[Dict[str, Any]]
     external_id: Optional[str]
     responses: Optional[List[UserResponseCreate]]
     suggestions: Optional[List[SuggestionCreate]]
     vectors: Optional[Dict[str, List[float]]]
 
+    @validator("fields")
+    @classmethod
+    def validate_chat_fields(cls, fields):
+        for key, value in fields.items():
+            if isinstance(value, list) and all(isinstance(item, ChatFieldValue) for item in value):
+                fields[key] = [msg.dict() for msg in value]
+        return fields
```

Comment:
I guess you are doing this so it's serializable when we are storing the record.

Instead of doing this we usually use `jsonable_encoder` on the context functions so we can still have `ChatFieldValue` instances (instead of dicts) when the data is parsed in the handler.

An example of doing this is inside the `create_response` context function:

```python
        response = await Response.create(
            db,
            values=jsonable_encoder(response_create.values),
            status=response_create.status,
            record_id=record.id,
            user_id=user.id,
            autocommit=False,
        )
```

Reply:
I can't get `jsonable_encoder` to encode a `List[ChatFieldValue]` do you have a suggestion?

Reply:
It should work if you try to execute `jsonable_encoder` to `fields` attribute. If you are not making in works I can try myself.

Reply:
Yeah I tried that, and ran out of time.

Reply:
No problem. I will take a look to it.